README.md

```diff
@@ -18,7 +18,7 @@ Se estiver usando Docker, os comandos para carregar o dump são:
 
 ```bash
 # Copiar o dump para a pasta temporária do Docker
-docker exec -it cp backup.sql POSTGRES_CONTAINER_ID:/tmp/backup.sql
+docker exec -it cp dev_dump.sql POSTGRES_CONTAINER_ID:/tmp/backup.sql
```

Comment:
this command doesn't work, as reviewd by [brunousml](https://github.com/brunousml) in the PR https://github.com/SOS-RS/backend/pull/91